modules/dr-restoring-cluster-state.adoc

```diff
@@ -281,75 +281,61 @@ etcd-ip-10-0-143-125.ec2.internal                1/1     Running     1
 +
 [source,terminal]
 ----
-$ oc delete node <non-recovery-controlplane-host-1> <non-recovery-controlplane-host-2>
+$ oc -n openshift-ovn-kubernetes delete pod -l app=ovnkube-control-plane
 ----
-
-. Verify that the Cluster Network Operator (CNO) redeploys the OVN-Kubernetes control plane and that it no longer references the non-recovery controller IP addresses. To verify this result, regularly check the output of the following command. Wait until it returns an empty result before you proceed to restart the Open Virtual Network (OVN) Kubernetes pods on all of the hosts in the next step.
+.. Deploy all of the ovnkube-controlplane pods again by running the following command:
 +
 [source,terminal]
 ----
-$ oc -n openshift-ovn-kubernetes get ds/ovnkube-master -o yaml | grep -E '<non-recovery_controller_ip_1>|<non-recovery_controller_ip_2>'
+$ oc -n openshift-ovn-kubernetes get pod -l app=ovnkube-control-plane
 ----
-+
-[NOTE]
-====
-It can take at least 5-10 minutes for the OVN-Kubernetes control plane to be redeployed and the previous command to return empty output.
-====
 
-. Restart the Open Virtual Network (OVN) Kubernetes pods on all the hosts.
+. If you are using OVNKubernetes network plugin, restart the Open Virtual Network (OVN) Kubernetes pods on all the nodes one by one. Steps to restart OVN-K pods in each node are as follows:
 +
 [NOTE]
 ====
-Validating and mutating admission webhooks can reject pods. If you add any additional webhooks with the `failurePolicy` set to `Fail`, then they can reject pods and the restoration process can fail. You can avoid this by saving and deleting webhooks while restoring the cluster state. After the cluster state is restored successfully, you can enable the webhooks again.
-
-Alternatively, you can temporarily set the `failurePolicy` to `Ignore` while restoring the cluster state. After the cluster state is restored successfully, you can set the `failurePolicy` to `Fail`.
+Restart OVN-K pods in the following order: the recovery control plane host, the other control plane hosts (if available), and lastly the other nodes.
 ====
 
 .. Remove the northbound database (nbdb) and southbound database (sbdb). Access the recovery host and the remaining control plane nodes by using Secure Shell (SSH) and run the following command:
 +
 [source,terminal]
 ----
-$ sudo rm -f /var/lib/ovn/etc/*.db
+$ sudo rm -f /var/lib/ovn-ic/etc/*.db
 ----
 
-.. Delete all OVN-Kubernetes control plane pods by running the following command:
+.. Restart the OpenVSwitch services. Access the node by using Secure Shell (SSH) and run the following command:
 +
 [source,terminal]
 ----
-$ oc delete pods -l app=ovnkube-master -n openshift-ovn-kubernetes
+$ sudo systemctl restart ovs-vswitchd ovsdb-server
 ----
 
-.. Ensure that any OVN-Kubernetes control plane pods are deployed again and are in a `Running` state by running the following command:
+.. Delete the `ovnkube-node` pod running on the node by running the following command:
```

Comment:
This uses "running" twice in the same sentence, with two different contexts. I would suggest rewording to avoid that. You can probably just remove the first instance of "running" and it would keep the same meaning